<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectVim</class>
 <widget class="QWidget" name="SelectVim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>931</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="empRb">
        <property name="text">
         <string>员工</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="attRb">
        <property name="text">
         <string>考勤</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectBt">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
